src/feed.js

```diff
@@ -326,7 +334,11 @@ class Feed {
       }
 
       if(entry.guid) {
-        item.push({ guid: entry.guid });
+        if (entry.guid.indexOf('http') === -1) {
```

Comment:
You may want to use [`startsWith`](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/String/startsWith) instead of `indexOf`.